[#1232] update readme (#1248)
dbs-leipzig/gradoop: README.md

The EPGM provides operators for both single logical graphs as well as collections 
of logical graphs; operators may also return single graphs or graph collections. 

> An overview of the implemented operators can be found [here](https://github.com/dbs-leipzig/gradoop/wiki/List-of-Operators).

Reviewer: ```suggestion
An overview and detailed descriptions of the implemented operators can be found in the [Gradoop Wiki](https://github.com/dbs-leipzig/gradoop/wiki/List-of-Operators).
```

#### Unary logical graph operators (one graph as input):

| Operator                    | Output | Output description                                           | Impl |
|:----------------------------|:-------|:-------------------------------------------------------------|:----:|
| Aggregation                 | G      | Graph with result of an aggregate function as a new property | Yes  |

> | Matching                    | GC     | Graphs that match a given graph pattern                      | Yes  |

Reviewer: I would split Matching in "Cypher" and "Query" to retain the naming of the operators itself.
Cypher: Graphs that match a given Cypher graph pattern
Query: Graphs that match a given Cypher-based GDL graph pattern

| Transformation              | G      | Graph with transformed (graph, vertex, edge) data            | Yes  |
| Grouping                    | G      | Structural condense of the input graph                       | Yes  |
| Subgraph                    | G      | Subgraph that fulfils given vertex and edge predicates       | Yes  |
| Cloning                     | G      | Creates a copy of a logical graph.                           | Yes  |

> | PageRankSampling            | G      | Creates a PageRank-Sampling of a logical graph.              | Yes  |

Reviewer: I would condens all samplings to a single "Sampling" operator.

| ClusteringCoefficient       | G      | Calculates the local/ global clustering coefficient.         | Yes  |
| WeaklyConnectedComponents   | G/GC   | Calculates or annotates weakly connected components using Flink&trade; Gelly.         | Yes  |
| HITS                        | G      | Operator wrapping Flink&trade; Gelly's Hyperlink-Induced Topic Search.                | Yes  |
| LabelPropagation            | G      | Operator executing label propagation integrated in Flink&trade; Gelly. | Yes  |
| PageRank                    | G      | Operator wrapping PageRank integrated in Flink&trade; Gelly.      | Yes  |
| RandomJump                  | G      | Performs RandomJump, using VertexCentricIteration Flink&trade; Gelly. | Yes  |
| SingleSourceShortestPaths   | G      | Operator wrapping SingleSourceShortestPaths integrated in Flink&trade; Gelly. | Yes  |
| TriangleCounting            | G      | Operator wrapping TriangleCounting integrated in Flink&trade; Gelly. | Yes  |
| DistinctVertexDegrees       | G      | Operator VertexDegrees calculation integrated in Flink&trade; Gelly. | Yes  |

> | PropertyExtraction          | G      | Extracts a property of a vertex.                             | Yes  |

Reviewer: ```suggestion
| PropertyExtraction          | G      | Extracts a property of a vertex to a new vertex.                             | Yes  |
```

> | ConnectNeighbors            | G      | Connects neighbors of a vertex.                              | Yes  |

Reviewer: ```suggestion
| ConnectNeighbors            | G      | Connects neighbors of a vertex with newly created edges.                              | Yes  |
```

| EdgeToVertex                | G      | Converts an edge to a vertex.                                | Yes  |
| VertexToEdge                | G      | Converts a vertex to an edge.                                | Yes  |

> | RollUp                      | GC     | Generates combinations according to the rollup operation in SQL. | Yes  |

Reviewer: I would move RollUp below "Grouping" because it is related. Change output description to: Graphs representig structural condense of the input graph according to SQL RollUp

| Split                       | GC     | Splits a logical graph into a GraphCollection. | Yes  |

> | BusinessTransactionGraphs   | GC     | EPGMVertex-centric implementation to isolate business transaction graphs. | Yes  |

Reviewer: This can be deleted.

> implementation which is used in Flink&trade;. The persistent representation of the EPGM

Reviewer: ```suggestion
implementation which is used in Flink&reg;. The persistent representation of the EPGM
```

is also contained in gradoop-common and together with its mapping to HBase&trade;.

### gradoop-data-integration

> This module contains graph importers (e.g from CSV files) and transformations on graphs (e.g

Reviewer: ```
Provides functionalities to support graph data integration. This includes minimal CSV and JSON importers as well as graph transformation operators (e.g. connect neighbors or conversion of edges to vertices and vice versa).
```
